backend/de.metas.business/src/main/java/de/metas/project/service/ProjectRepository.java

```diff
@@ -144,39 +130,15 @@ public ImmutableSet<ProjectId> getProjectIdsDownStream(@NonNull final Collection
 				rs -> ProjectId.ofRepoId(rs.getInt(1)));
 	}
 
-	public void updateFromProjectType(@NonNull final I_C_Project projectRecord)
-	{
-		final ProjectTypeId projectTypeId = ProjectTypeId.ofRepoIdOrNull(projectRecord.getC_ProjectType_ID());
-		if (projectTypeId == null)
-		{
-			return;
-		}
-
-		final String projectValue = computeNextProjectValue(projectRecord);
-		if (projectValue != null)
-		{
-			projectRecord.setValue(projectValue);
-		}
-		if (Check.isEmpty(projectRecord.getName()))
-		{
-			projectRecord.setName(projectValue != null ? projectValue : ".");
-		}
-
-		final ProjectType projectType = projectTypeRepository.getById(projectTypeId);
-		projectRecord.setProjectCategory(projectType.getProjectCategory().getCode());
-	}
-
 	@NonNull
 	private Project save(
 			@NonNull final I_C_Project projectRecord,
 			@NonNull final ProjectData projectData)
```

Comment:
i have the feeling projectData has the same problem as budgetProjectData

Reply:
what's the resolution here?

Reply:
out of scope :-s ....

Reply:
true. 
but pls, let's keep in mind next time to not build more on top of ProjectData. 
We would need to refactor it sooner or later.